### PR TITLE
revert cro

### PR DESCRIPTION
**Pull request by briangallagher**
# Description
revert cro to 0.24.1 to postpone the RDS change from 10.13 to 10.15

# Verify
Run the upgrade from 1.6.1 to this branch on a ccs cluster.
Verify RDS version 10.13
Sanity regression testing in particular around CRO

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,7 +26,7 @@ require (
 	github.com/headzoo/surf v1.0.0
 	github.com/headzoo/ut v0.0.0-20181013193318-a13b5a7a02ca // indirect
 	github.com/integr8ly/application-monitoring-operator v1.6.0
-	github.com/integr8ly/cloud-resource-operator v0.25.0
+	github.com/integr8ly/cloud-resource-operator v0.24.0
```
Comment (pmccarthy) [inline go.mod:29]: Should this be v0.24.1?

File: go.sum [hunk 2]
```diff
@@ -717,6 +717,7 @@ github.com/influxdata/tdigest v0.0.0-20181121200506-bf2b5ad3c0a9/go.mod h1:Js0mq
 github.com/influxdata/usage-client v0.0.0-20160829180054-6d3895376368/go.mod h1:Wbbw6tYNvwa5dlB6304Sd+82Z3f7PmVZHVKU637d4po=
 github.com/integr8ly/application-monitoring-operator v1.6.0 h1:vQV5vXxOho2lONcJe0qTV+oxg7qWE2l4hee4ACtS620=
 github.com/integr8ly/application-monitoring-operator v1.6.0/go.mod h1:08svhH7y8+X89PDDZESLsRAg1+uMJU7IM78Q+03wdOY=
+github.com/integr8ly/cloud-resource-operator v0.24.0/go.mod h1:pxmR7H2T5MJqcw1F9DTmsX3YfEc419Q/tp5m2g8xdZk=
```
[the remark this answers is not in the document]
Comment (briangallagher) [reply go.sum:720]: same as above

[the remark this answers is not in the document]
Comment (MStokluska) [reply go.sum:721]: @briangallagher the checksum needs to be updated to 9xJ0yOK5e30lqrN7NRK/ualuJeiAdLt8eDKxl0R3iHw=

**Pull request closed without merging**